ui/lib/kv/addon/components/page/secrets/create.hbs

```diff
@@ -9,4 +9,27 @@
   </p.levelLeft>
 </PageHeader>
 
-<hr class="is-marginless has-background-gray-200" />
\ No newline at end of file
+<Toolbar>
+  <ToolbarFilters>
+    <Toggle
+      data-test-json-view-toggle
+      @name="json"
+      @status="success"
+      @size="small"
+      @checked={{this.showJsonView}}
+      @onChange={{this.toggleJsonView}}
+    >
+      <span class="has-text-grey">JSON</span>
+    </Toggle>
+  </ToolbarFilters>
+</Toolbar>
+
+<hr class="is-marginless has-background-gray-200" />
+
+<KvObjectEditor
+  class="has-top-margin-m"
+  data-test-kv={{@secret.secretData}}
```

Comment:
is secretData going to be an object? so the data-test name here would be a little strange?

Reply:
Yeah - this was just copy pasted from somewhere else and so I updated to match what those values were - the data test selector would change here